packages/server/graphql/getValidTeamIds.ts

```diff
@@ -0,0 +1,13 @@
+import {DataLoaderWorker} from './graphql'
+
+export const getValidTeamIds = async (
```

Comment:
-1 `getAccessibleTeamIds` or `filterByTeamMemberIds` would be more descriptive. All input ids could be "valid" in other contexts.